### ISSUE TITLE
███ ~ README ~ THE PROJECT IS NO LONGER DEVELOPED ~ README ~ ███

### ISSUE THREAD

**Issue by kulfoon** [reek/anti-adblock-killer#4179]
This project is not being developed anymore, don't post here anymore, you can still post your issues in the following projects which are alive:

- uBlock Origin: [the website](https://en.wikipedia.org/wiki/UBlock_Origin) / [the filters issue tracker](https://github.com/uBlockOrigin/uAssets/issues)

- Nano Defender (continuation of AAK-REEK): [the website](https://jspenguin2017.github.io/uBlockProtector) / [the filters issue tracker](https://github.com/NanoAdblocker/NanoFilters/issues) 

Also before posting your issue you can try if adding these filter lists solve your issue:

- [Adblock Warning Removal List](https://easylist.to/pages/other-supplementary-filter-lists-and-easylist-variants.html#adblock-warning-removal-list)
- [Fanboy's Enhanced Trackers List](https://www.fanboy.co.nz/filters.html)

**Comment by uBlock-user** [comment]
Already posted in the wiki - https://github.com/reek/anti-adblock-killer/wiki/Nano-Defender, https://github.com/reek/anti-adblock-killer/wiki/AAK-Dead-and-Discontinued

They don't care, they don't read.

**Comment by kulfoon** [comment]
I know, but as for now it seems that they began to read and create issues on uBO and ND filters issue trackers, as some new nicks & issues can be seen on the both trackers last 3 days, hope they will keep going like that, we'll see for how long...

**Comment by uBlock-user** [comment]
It's a losing battle man, you're not the first who made a dedicated post about this(you can search the tracker for this), people always come and they always without looking, that's just the way around here.

**Comment by pomidoor** [comment]
Yea, I know, not all of them are curable, but at least some of these jackasses have been redirected to proper issue trackers, better this than nothing.

**Comment by kulfoon** [comment]
Yea, I know, not all of them are curable, but at least some of these jackasses have been redirected to proper issue trackers, better than nothing.

**Comment by gargul03** [comment]
Yep, they are like a herd of wild buffaloes ramming everything in their path. :smiley:

**Comment by gargul04** [comment]
Yep, they are like a herd of wild rushing buffalo ramming everything in their path

**Comment by garry-ut99** [comment]
Yep, they are like a herd of wild rushing buffalo ramming everything in their path.
